src/metabase/api/common/internal.clj

```diff
@@ -290,7 +328,7 @@
                                        (-> schema
                                            (mc/explain value)
                                            me/with-spell-checking
-                                           me/humanize)}}))))
+                                           (me/humanize {:wrap mu/humanize-include-value}))}}))))
```

Comment:
This will include the failing value in the humanized explaination.

e.g. `"value must be an integer greater than zero., received: \"diamond\""`